assets/src/edit-story/components/library/panes/text/textTab.js

```diff
@@ -82,8 +81,7 @@ function TextTab(props) {
   const handleAddText = (evt) => {
     evt.stopPropagation();
     insertElement('text', {
-      content: __('Fill in some text', 'web-stories'),
-      fontSize: DEFAULT_FONT_SIZE,
+      ...DEFAULT_FONT('default-text'),
```

Comment:
Is it a function?

Reply:
Sorry about that, I missed that from when I initially made it a function.